tests/third_party/cupy/linalg_tests/test_solve.py

```diff
@@ -47,7 +47,6 @@ def check_x(self, a_shape, b_shape, xp, dtype):
         testing.assert_array_equal(b_copy, b)
         return result
 
-    @pytest.mark.skipif(is_cpu_device(), reason="SAT-6842")
```

Comment:
`tests/test_sycl_queue.py::test_solve` is muted with the same reason
Should the skip mark be removed also there?
Otherwise probably we need to update the reason there...

Reply:
Yes, I am going to do it in a separate PR since not all functions with this skip mark work (tests solve works)